bin/onchange-db.sh

```diff
@@ -5,5 +5,5 @@
 consul-template \
     -once \
     -dedup \
-    -consul ${CONSUL}:8500 \
+    -consul-addr ${CONSUL}:8500 \
```

Comment:
The latest version of consul-template was throwing warnings/notices about the change to the argument